src/transport_ofi.h

```diff
@@ -150,37 +152,37 @@ void shmem_transport_ofi_drain_cq(void)
 				              (void *)&e, 0);
 				RAISE_ERROR(e.err);
 			} else {
-				RAISE_ERROR(ret);
-		}
+				OFI_RET_CHECK(ret);
+			}
 		}
 
 		if(ret == 1) {
-	shmem_transport_ofi_frag_t *frag =
-			container_of(buf.op_context,
-			struct shmem_transport_ofi_frag_t,
-		     context);
```

Comment:
It's a good practice to isolate whitespace changes to separate commits.  Please don't bother making a change for this PR.